src/partials/products-section.html

```diff
@@ -4,28 +4,43 @@
     <h2 class="products__title">PRODUCTS</h2>
     <ul class="products__list">
       <li class="products__list-item">
-        <h3>ice cream</h3>
-        <div>...</div>
-        <p>
+        <h3 class="products__subtitle">ice cream</h3>
+        <div class="products__suspension-dots"></div>
+        <p class="products__description">
           Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
           eiusmod tempor incididunt ut labore et
         </p>
+        <button class="products__btn" type="button">
```

Comment:
aizi ar trebuio sa fie independent butonul ca mai apare si in alte sectiuni gen arrow-btn si folositi o singura clasa peste tot si modificator daca se scimba ceva la careva